internal/handlers/sqlite/msg_explain.go

```diff
@@ -83,6 +85,43 @@ func (h *Handler) MsgExplain(ctx context.Context, msg *wire.OpMsg) (*wire.OpMsg,
 		qp.Filter = params.Filter
 	}
 
+	if h.EnableSortPushdown {
+		iter := params.Sort.Iterator()
+		defer iter.Close()
+
+		var key string
+		var order types.SortType
+
+		for {
```

Comment:
`params.Sort.Len()` is either 1 or not 1. There is no need to iterate at all

Reply:
Do you mean not use iterator at all and get values directly like this?

```
	k := params.Sort.Keys()[0]
	v := must.NotFail(params.Sort.Get(k))
```

Reply:
or `.Values()[0]`

Reply:
True, that is simpler :+1:

@noisersup let's do that :hugs: